docs/reference/migration/apis/deprecation.asciidoc

```diff
@@ -6,6 +6,11 @@
 <titleabbrev>Deprecation info</titleabbrev>
 ++++
 
+IMPORTANT: Before performing a major version upgrade, make sure you have checked
+this API using the most recent minor version of the major version you are
+upgrading from. In earlier minor versions, this API may be missing information
+required for the major version upgrade.
+
```

Comment:
This might be *slightly* less awkward (it's confusing, no matter what):

> Use this API to check for deprecations before performing a major version upgrade. You should run it on
the last minor version of the major version you are upgrading from. Earlier minor versions might
not include all deprecations.

Reply:
Thanks! This is clearer, I've changed it to use a (very slightly tweaked) version of this.